README.md

```diff
@@ -38,12 +38,14 @@ An RFC can be merged when it receives an absolute majority of approvals from [Ma
 
 You can also ask for help with the RFC process on Slack at [#ref-help](https://govau-guides.slack.com/archives/C02B0046A5T).
 
-An RFC can be merged once it has an absolute majority of PR approvals of all maintainers.
-
-Make sure you check the [list of maintainers](https://github.com/orgs/designsystemau/teams/maintainers), since the number of maintainers may vary.
+An RFC can be merged once it has 5 PR approvals from [Maintainers](https://github.com/orgs/designsystemau/teams/maintainers).
 
 ## Who will implement my RFC?
 
 Anyone may pick up an RFC and ask to implement it.
 
 To raise your hands to implement an RFC submit a PR to the RFC document and add your name to the `Implementer: ` section at the top.
+
+### Project management
+
+Work should be tracked and assigned clearly at a new GitHub Projects board. 
```

Comment:
Does this mean we have a board for each task? I agree we should have a clear way to see who works on what and where. A single board that shows the stages for each RFC is a great idea.
Sorta like this:

| Proposal | Approved | In Progress | Done |
|----------|-----------|---|---|
| `RFC7`  | `RFC3` | `RFC2` | `RFC1` | `RFC5`|
| | | `RFC6` | | |

Reply:
One level lower: one Board per action-ready RFC

Reply:
I created this to capture your suggestion https://github.com/designsystemau/RFCs/projects/1

Reply:
I like this!